### PR TITLE
Index page: header, footer and mithril example

### PR DESCRIPTION
**Pull request by jomei**
#4

### CODE REVIEW

File: web/static/css/app.styl [hunk 1]
```diff
@@ -1 +1,16 @@
...
+@import fa-dir-path + "/stylus/index.styl"
+
+.inline-block
+  display inline-block
+
+.center-block
+  margin auto
+
+main
+  min-height 700px
```
Comment (begor) [inline web/static/css/app.styl]: You definitely should read "Code Complete" to avoid such a silly mistakes.

File: web/static/js/app.coffee [hunk 2]
```diff
@@ -18,4 +18,35 @@
...
+    @text = m.prop data.text
+
+  @list: () ->
+    # fetch data from server and build list of Foos
+    [new CB.model.Foo(text: 'Change me, ept'), new CB.model.Foo(text: 'second Foo')]
+
+class CB.Demo
+  @controller: ->
+    foos = CB.model.Foo.list()
+    foos: foos
```
Comment (begor) [inline web/static/js/app.coffee]: Why we can't simply
`foos: CB.model.Foo.list()`
?

File: web/static/js/app.coffee [hunk 3]
```diff
@@ -18,4 +18,35 @@
...
+      m('.foo-item', [
+        m('input[type="text"]', {oninput: m.withAttr('value', foo.text), value: foo.text()})
+        m('span', foo.text())
+      ])
+    )
+
+
+
+do($, m) ->
+  m.route(document.getElementById('test-container'), '/',
```
Comment (begor) [inline web/static/js/app.coffee]: We don't use jQuery here and I think we would be fine without it all the way.
So, is there some kind of minimalist Bootstrap build, without jQuery-like deps?

**Comment by begor** [comment]
Can you please actualize useless test for index page? 
Failing build looks pretty sad.

File: web/static/css/app.styl (hunk 1 above)
Comment (jomei) [inline web/static/css/app.styl]: ?

File: web/static/js/app.coffee (hunk 2 above)
Comment (jomei) [inline web/static/js/app.coffee]: of cource we can, but it just example and it will be remove in next task

File: web/static/css/app.styl (hunk 1 above)
Comment (begor) [inline web/static/css/app.styl]: You forgot an empty line...

Comment (jomei) [inline web/static/css/app.styl]: oh jeez....

**Comment by jomei** [comment]
@begor , fixed

**Pull request merged**